src/app/pages/SearchResultsPage/__tests__/SearchResultsView.test.tsx

```diff
@@ -63,7 +63,7 @@ describe('SearchResultsView', () => {
     expect(screen.getByRole('link', { name: '337325.43836 FTP' })).toBeInTheDocument()
     expect(screen.getByRole('link', { name: '337325.43836 FTP' })).toHaveAttribute(
       'href',
-      '/emerald/account/oasis1qrvha284gfztn7wwq6z50c86ceu28jp7csqhpx9t/tokens/erc-20#oasis1qpssvkplnlpzdvwxpgmrhf9j5lkyvaylcvujhjhg',
+      '/emerald/account/0xBA504818FdD8D3dBA2Ef8fD9B4F4D5c71aD1d1D3/tokens/erc-20#oasis1qpssvkplnlpzdvwxpgmrhf9j5lkyvaylcvujhjhg',
       // TODO: use evm account address
       // '/emerald/account/0xBA504818FdD8D3dBA2Ef8fD9B4F4D5c71aD1d1D3/tokens/erc-20#oasis1qpssvkplnlpzdvwxpgmrhf9j5lkyvaylcvujhjhg',
```

Comment:
cleanup

Reply:
> cleanup

Oops I missed this before merging. Finished in https://github.com/oasisprotocol/explorer/pull/248.